modules/agent-configuration-parameters.adoc

```diff
@@ -62,7 +62,9 @@ If this address is not provided, one IP address is selected from the provided ho
 |IPv4 or IPv6 address.
 
 |bootArtifactsBaseURL:
-|The URL of the server to upload Preboot Execution Environment (PXE) assets to when using the Agent-based Installer to generate an iPXE script.
+|When using the Agent-based Installer to generate a minimal ISO image, specifies a URL where the rootfs image file can be retrieved from during cluster installation. This is optional for booting minimal ISO images in connected environments.
```

Comment:
Good point, thanks! Though I'm not fully sure about rootfs being in backticks. It's like the general name of the file, but does not necessary refer to the explicit file name, and looking around the docs it does not seem like something that's presented in backtick.

It may be worth looking into in a separate effort though, in order to ensure consistency